keras/src/backend/config.py

```diff
@@ -15,6 +15,9 @@
 # Default backend: TensorFlow.
 _BACKEND = "tensorflow"
 
+# Default for NNX backend features.
+_NNX_ENABLED_KEY = "nnx_enabled"
```

Comment:
probably just get rid of this? we aren't tracking our other json keys as constants, doing it for only one is just confusing